rtengine/procparams.h

```diff
@@ -452,7 +452,7 @@ struct ColorToningParams {
     double labgridBLow;
     double labgridAHigh;
     double labgridBHigh;
-    static constexpr double LABGRID_CORR_MAX = 8000.f;
```

Comment:
@agriggio I understand, this wasn't the best for evaluation, but code-wise it would be nice to change it back to the `constexpr` when you are finished.

Thanks,
Flössie